Documentation/HowToGuides/CreatingProxyActionsAndChainingActions/README.md

```diff
@@ -0,0 +1,82 @@
+# Creating Proxy Actions And Chaining Actions
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+It is sometimes desirable to have many different input actions all have the same outcome. It is possible to simply do this by wiring up all of the events in all of the actions to have the same output, however this can get cumbersome and hard to maintain.
+
+The Zinnia Actions can be used as proxies to enable chaining of Actions to all drive the same outcome defined in one place.
+
+Following on from the scene created in Adding A Unity Button Action, we’re going to move the Action functionality to a Proxy Action and then chain the button press Action and keyboard Action to this newly created Proxy Action. This will move our output functionality into one place and allow us to call it from many different actions.
```

Comment:
Shouldn't be `Adding A Unity Button Action` as the other guide changed and is now `Using The Unity Button Action`

Also, probably worth just making this the link by wrapping it in `[]`

like:

`Following on from the scene created in [Using The Unity Button Action]`

Because the link is already defined at the bottom, you don't need to define it again.